src/tests/ftest/security/pool_security_acl.py

```diff
@@ -0,0 +1,403 @@
+
+    def pool_acl_verification(self, current_user_acl, read, write):
+        '''
+        Deascription:
+            Daos pool security verification with acl file.
+            Steps:
+                (1)Setup dmg tool for creating a pool
+                (2)Generate acl file with permissions
+                (3)Create a pool with acl
+                (4)Verify the pool create status
+                (5)Get the pool's acl list
+                (6)Verify pool read operation
+                (7)Verify pool write operation
+                (8)Cleanup user and destroy pool
+        Args:
+            current_user_acl: acl with read write access credential.
+            read: expecting read permission.
+            write: expecting write permission.
+        Return:
+            pass to continue.
+            fail to report the testlog and stop.
+        '''
+
+        # (1)Create daos_shell command
+        dmg = DmgCommand(os.path.join(self.prefix, "bin"))
+        dmg.get_params(self)
+        port = self.params.get("port", "/run/server_config/*", 10001)
+        get_acl_file = self.params.get("acl_file", "/run/pool_acl/*",
+                                       "acl_test.txt")
+        acl_file = os.path.join(self.tmp, get_acl_file)
+        num_user = self.params.get("num_user", "/run/pool_acl/*")
+        num_group = self.params.get("num_group", "/run/pool_acl/*")
+        servers_with_ports = [
+            "{}:{}".format(host, port) for host in self.hostlist_servers]
+        dmg.hostlist.update(",".join(servers_with_ports), "dmg.hostlist")
+        self.log.info("  (1)dmg= {}".format(dmg))
+
+        # (2)Generate acl file with permissions
+        self.log.info("  (2)Generate acl file with user/group permissions")
+        permission_list = self.create_pool_acl(num_user,
+                                               num_group,
+                                               current_user_acl,
+                                               acl_file)
+
+        # (3)Create a pool with acl
+        self.log.info("  (3)Create a pool with acl")
+        dmg.action_command.acl_file.value = acl_file
+        dmg.exit_status_exception = False
+        result = dmg.run()
+
+        # (4)Verify the pool create status
+        self.log.info("  (4)dmg.run() result=\n{}".format(result))
+        if result.stderr == "":
+            uuid, svc = dmg_utils.get_pool_uuid_from_stdout(result.stdout)
+        else:
+            self.fail("##(4)Unable to parse pool uuid and svc.")
+
+        # (5)Get the pool's acl list
+        #    dmg pool get-acl --pool <UUID>
+        self.log.info("  (5)Get a pool's acl list by: "
+                      "dmg pool get-acl --pool --hostlist")
+        pool_acl_list = self.get_pool_acl_list(uuid)
+        self.log.info(
+            "   pool original permission_list: {}".format(permission_list))
+        self.log.info(
+            "   pool get_acl  permission_list: {}".format(pool_acl_list))
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments